src/linker/Linker/LinkContext.cs

```diff
@@ -531,6 +537,12 @@ public void LogWarning (string text, int code, MessageOrigin origin, string subc
 			if (!LogMessages)
 				return;
 
+			if ((GeneralWarnAsError && !WarnAsError.Contains (((uint) code, false))) ||
+				(!GeneralWarnAsError && WarnAsError.Contains (((uint) code, true)))) {
+				LogError (text, code, subcategory, origin, true);
```

Comment:
```suggestion
				LogError (text, code, subcategory, origin, isWarnAsError: true);
```

For documentation purposes